docs/installation.rst

```diff
@@ -75,8 +75,7 @@ Troubleshooting
 
 Some setups (like MacOS) have trouble with building the dependencies required
 for reading SSL certificates.  If you don't care about SSL stuff and only want
-to use sagan to say, parse traceroute or DNS results, then you can tell the
-installer to skip building ``pyOpenSSL`` by doing the following::
+to use sagan to say, parse traceroute or DNS results, then you can do the following:
```

Comment:
This may require additional explanation or none at all and probably needs a little experimentation on different platforms to be sure.  `PyOpenSSL` was a pain in the ass to install on a Mac, but I'm not sure if that's also the case with `cryptography`.  I know that in Linux environments, `cryptography` requires certain system-level libraries to be installed, and I'm sure that Mac & Windows each have their own quirks.

The good news however is that this is all beautifully documented by the `cryptography` team [here](https://cryptography.io/en/latest/installation/) so you might want to adjust this paragraph to just give the user a heads-up and point them to that URL if they have trouble.